src/app/Components/ComponentsSamples/ContextMenus/ContextMenuSamples.xaml

```diff
@@ -0,0 +1,163 @@
+                        <dui:ContextMenuItem Title="Item 2" />
+                    </dui:ContextMenuGroup>
+                </dui:ContextMenu>    
+            </dui:ContextMenuEffect.ItemsSource>
+        </dui:Button>
+        
+        <!--Checkable items (single)-->
+        <dui:Button Text="Checkable items (single)"
+                    dui:ContextMenuEffect.HasContextMenu="True">
+            <dui:ContextMenuEffect.ItemsSource>
+                <dui:ContextMenu VerticalOptions="Start"
+                                 ItemClickedCommand="{Binding ItemClickedCommand}">
+                    <dui:ContextMenuGroup IsCheckable="True">
+                        <dui:ContextMenuItem Title="Item 1" />
+                        <dui:ContextMenuItem Title="Item 2" />
+                    </dui:ContextMenuGroup>
+                </dui:ContextMenu>    
+            </dui:ContextMenuEffect.ItemsSource>
+        </dui:Button>
+
+        <!-- Grouped items -->
+        <dui:Button Text="Grouped items"
+                    dui:ContextMenuEffect.HasContextMenu="True">
+            <dui:ContextMenuEffect.ItemsSource>
+                <dui:ContextMenu VerticalOptions="Start"
+                                 ItemClickedCommand="{Binding ItemClickedCommand}">
+                    <dui:ContextMenuGroup Title="Group 1">
+                        <dui:ContextMenuItem Title="Item 1" />
+                        <dui:ContextMenuItem Title="Item 2" />
+                    </dui:ContextMenuGroup>
+                    <dui:ContextMenuGroup Title="Group 2">
+                        <dui:ContextMenuItem Title="Item 1" />
+                        <dui:ContextMenuItem Title="Item 2" />
+                    </dui:ContextMenuGroup>
+                </dui:ContextMenu>    
+            </dui:ContextMenuEffect.ItemsSource>
+        </dui:Button>
+
+        <!-- Checkable grouped items -->
+        <dui:Button Text="Checkable grouped items"
+                    dui:ContextMenuEffect.HasContextMenu="True">
+            <dui:ContextMenuEffect.ItemsSource>
+                <dui:ContextMenu VerticalOptions="Start"
+                                 ItemClickedCommand="{Binding ItemClickedCommand}">
+                    <dui:ContextMenuGroup Title="Group 1"
+                                          IsCheckable="True">
+                        <dui:ContextMenuItem Title="Item 1" />
+                        <dui:ContextMenuItem Title="Item 2" />
+                    </dui:ContextMenuGroup>
+                    <dui:ContextMenuGroup Title="Group 2"
+                                          IsCheckable="True">
+                        <dui:ContextMenuItem Title="Item 1" />
+                        <dui:ContextMenuItem Title="Item 2" />
+                    </dui:ContextMenuGroup>
+                </dui:ContextMenu>    
+            </dui:ContextMenuEffect.ItemsSource>
+        </dui:Button>
+        
+        <Frame BackgroundColor="{dui:Colors color_primary_d_90}"
```

Comment:
Du kan fjerne denne, den er ikke så interessant. Jeg skjønner at du brukte det for å teste, men den er ikke nødvendig.